platform-stack/docker-compose.yml

```diff
@@ -117,22 +113,22 @@ services:
       - local-keycloak
 
   postgres:
-    image: postgres:13.2
+    image: postgres:15.3
     container_name: postgres
     volumes:
-      - ./postgres:/docker-entrypoint-initdb.d
+      - ./postgres/:/docker-entrypoint-initdb.d/
+      - ./data:/var/lib/postgresql/data
     ports:
       - "5432:5432"
     restart: unless-stopped
     healthcheck:
-      test: [ "CMD-SHELL", "psql -l | grep keycloak | wc -l" ]
+      test: [ "CMD-SHELL", "su - postgres -c 'psql -l | grep $${SSO_DATASOURCE_DB} | wc -l'" ]
       interval: 5s
       timeout: 5s
       retries: 5
     environment:
       - POSTGRES_MULTIPLE_DATABASES=${ENGINE_DATASOURCE_DB},${SCHEMA_DATASOURCE_DB},${SSO_DATASOURCE_DB},${S3INTEGRATION_DATASOURCE_DB}
       - POSTGRES_PASSWORD=${LOCAL_DATASOURCE_PASSWORD}
-      - POSTGRES_USER=${LOCAL_DATASOURCE_USERNAME}
```

Comment:
I'd leave this as is, since it's more explicit. If someone would like to work with another user, he/she would need to add this line again